docs/source/interfaces/muon/feature_flags.rst

```diff
@@ -0,0 +1,83 @@
+.. _Muon_Feature_Flags-ref:
+
+Muon Feature Flags
+==================
+
+.. contents:: Table of Contents
+  :local:
+
+Overview
+--------
+
+Muon Feature Flags provides super users with access to additional features such as raw plots and model analysis.
```

Comment:
I'm not sure which terminology we should go for. This really is only for a special group of users in very specific circumstances. @AnthonyLim23 what do you think we should use?

Reply:
We used the term super users in part because it was just scientists. Since it is going to be a documented and advertised feature, maybe we should consider changing it. Maybe we should just say users?

Reply:
Personally, given that you have to manually edit the `mantid.user.properties` file, I think "advanced users" would be the best term. It would dissuade regular users that don't need these features from messing around in a config file that can damage their settings